backend/lib/edgehog/tenants/tenants.ex

```diff
@@ -36,6 +36,12 @@ defmodule Edgehog.Tenants do
   end
 
   resources do
-    resource Edgehog.Tenants.Tenant
+    resource Edgehog.Tenants.Tenant do
+      define :create_tenant, action: :create
+      define :provision_tenant, action: :provision
+      define :fetch_tenant_by_slug, action: :by_slug, args: [:slug]
```

Comment:
Same here, you should be able to delete the `by_slug` action and rely on the default `read`.

```suggestion
      define :fetch_tenant_by_slug, action: :read, get_by: :slug, not_found_error?: true
```